platform/FEATURE_EXPERIMENTAL_API/FEATURE_PSA/TARGET_TFM/TARGET_TFM_LATEST/scripts/mbed_set_post_build_tfm.cmake

```diff
@@ -17,7 +17,7 @@ function(mbed_post_build_tfm_sign_image
     set(mbed_target_name ${mbed_target})
     set(post_build_command
         COMMAND ${Python3_EXECUTABLE}
-            ${MBED_PATH}/platform/FEATURE_EXPERIMENTAL_API/FEATURE_PSA/TARGET_TFM/TARGET_TFM_LATEST/scripts/generate_mbed_image.py
+            ${mbed-os_SOURCE_DIR}/platform/FEATURE_EXPERIMENTAL_API/FEATURE_PSA/TARGET_TFM/TARGET_TFM_LATEST/scripts/generate_mbed_image.py
```

Comment:
Can this not be reduced to `${CMAKE_CURRENT_LIST_DIR}/generate_mbed_image.py`?

Reply:
we are in the targets area, not in the tools so current list dir points to targets/ somewhere.

Reply:
Anyway, this path is incorrect. I'll fix it.